### PR TITLE
Sjofn hw6

### PR DESCRIPTION
**Pull request by tzatushevskaya**
- attempt for HW6

### CODE REVIEW

File: src/views/Sjofn/Homework6.vue [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  data () {
+    return {
+      ingredients: '',
+      dish: '',
+      isLoading: false,
+      results: []
+    }
+  },
+  computed: {
+    isSearchButtonDisabled () {
```
Comment (eclimov) [inline src/views/Sjofn/Homework6.vue:93]: This computed property is never used. Why did you add it then?
Please, either remove it or use it somewhere in the component

File: src/About.vue [hunk 2]
```diff
@@ -11,6 +11,9 @@
       <li>
         <a href="">Homework4</a>
       </li>
+      <li>
+        <a href="">Homework6</a>
+      </li>
```
Comment (eclimov) [inline src/About.vue]: The whole `About.vue` component looks redundant. 
As far as I remember, it is not used anywhere. Please, check and remove it, if it's true.

**Review by eclimov — APPROVED** [review]
Approved

**Review by SandaPanteleimonov — APPROVED** [review]
Approved

**Comment by eclimov** [comment]
Checked
